models/category.ts

```diff
@@ -0,0 +1,68 @@
+import { Category } from "../types/categories.js";
+
+export class CategoryRepo {
+  categories: Category[] = [
+    {
+      id: 1,
+      name: "Laptop",
+      description: "High-performance laptop for all your needs.",
+    },
+    {
+      id: 2,
+      name: "Smartphone",
+      description: "Latest smartphone with advanced features.",
+    },
+  ];
+
+  generateNewId() {
```

Comment:
Move this function to /utils/generateNewId.ts
I should not be method of this class

Reply:
Thanks, Done